papaya-core/src/main/java/pl/edu/agh/papaya/service/jira/JiraAuthorisationService.java

```diff
@@ -0,0 +1,82 @@
+package pl.edu.agh.papaya.service.jira;
+
+import com.google.api.client.auth.oauth.OAuthParameters;
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpRequestFactory;
+import com.google.api.client.http.HttpResponse;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.util.Value;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+import java.util.Optional;
+import java.util.Scanner;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.json.JSONArray;
+import org.springframework.stereotype.Service;
+
+@Service
+@Getter
+@Setter
+@NoArgsConstructor
+public class JiraAuthorisationService {
+
+    @Value("${jira.auth.private.key}")
+    private static String PRIVATE_KEY;
+    @Value("${jira.auth.consumer.key}")
+    private static String CONSUMER_KEY;
```

Comment:
`CONSUMER_KEY` to jest ten string `"OauthKey"` zawsze? Nie widzę potrzeby trzymania tego w zmiennej środowiskowej. Zrobiłabym z tego pole `private static final` - może nawet w `JiraOAuthClient`, to zmniejszymy trochę liczbę argumentów

Na zmienną środowiskową zasługuje klucz prywatny i publiczny na pewno (klucz publiczny można też mieć tu, żeby były w jednym miejscu, ale będzie potrzebny gdzie indziej)

Reply:
`CONSUMER_KEY` występuje teraz jako stała w `JiraOAuthClient`. Dodano nową zmienną środowiskową reprezentującą klucz publiczny